src/utils/partition_stl_containers.hpp

```diff
@@ -20,10 +20,11 @@
 
 namespace parthenon {
 namespace partition {
-// TODO(JMM): The templated type safety with T* and T may need to be
-// changed if we move to sufficiently general container objects.
+// Note the interface here has objects in partition are now
+// COPIED, not pointed at.
+// Change is due to using std::shared_ptr<MeshBlock>
```

Comment:
I now read the comment as: All elements (e.g., variables of a `Container`) are not copied into the Partition.
Is this correct?

Reply:
The partitioner now copies the element of the stl container into the partition. But the element is a pointer. Before, I had to access the pointer to the objects to avoid unnecessary copies.

Reply:
Gotcha!